l10n_jp_hr_employee/views/res_users_views.xml

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_users_view_form_profile" model="ir.ui.view">
+        <field name="name">res.users.preferences.form.inherit</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="hr.res_users_view_form_profile"/>
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button name="action_submit" type="object" attrs="{'invisible': ['|',('state','!=','draft'),('temp_save','!=',False)]}" string="Submit" class="oe_highlight"/>
+                <button name="action_confirm" type="object" states="submit" string="Confirm" class="oe_highlight" groups="hr.group_hr_manager"/>
+                <button name="action_draft" type="object" states="submit,confirm" string="Set to Draft"/>
+                <field name="state" widget="statusbar" clickable="0"/>
+            </xpath>
+            <xpath expr="//page[@name='personal_information']/group" position="before">
+                <div class="alert alert-info" role="alert">
```

Comment:
This should show only when employees are allowed to edit their own data, I suppose.